net-misc/sipcalc/sipcalc-1.1.6.ebuild

```diff
@@ -1,12 +1,13 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
 DESCRIPTION="Sipcalc is an advanced console-based IP subnet calculator"
-HOMEPAGE="http://www.routemeister.net/projects/sipcalc/"
+HOMEPAGE="http://www.routemeister.net/projects/sipcalc/
+	https://github.com/sii/sipcalc"
 SRC_URI="http://www.routemeister.net/projects/${PN}/files/${P}.tar.gz"
+
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="amd64 ~arm64 sparc x86 ~amd64-linux ~x86-linux"
```

Comment:
Let's please reset KEYWORDS due to unseen changes by default EAPI implementations. We can call stabilization a bit faster than 30 days though.
(and resetting keywords means -r1 bump)

Reply:
sure, no problem, i've updated my PR